doc/content/source/kernels/PotentialGradientSource.md

```diff
@@ -1,21 +1,33 @@
 # PotentialGradientSource
```

Comment:
Agreed, I will remove `PotentialGradientSource`, since the name `EFieldMagnitudeSource` will probably fit better with @cticenhour's  `FieldSolverMaterial` for electromagnetic coupling.